src/components/providers/DirectoryHeader.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+  Flex,
+  useTheme,
+  Text,
+  Tag,
+  TagLabel,
+  Container,
+} from "@chakra-ui/react";
+
+const DirectoryHeader = () => {
+  const theme = useTheme();
+
+  return (
+    <Flex
+      bgColor={theme.colors.neutral[0]}
+      px={{ base: "20px", sm: "20px", md: 20 }}
+      h={192}
+      placeContent="center"
+    >
+      <Container py="24px" px="8px">
+        <Text
+          color={theme.colors.neutral[900]}
+          fontWeight={500}
+          fontSize="24px"
+          my="5px"
+        >
+          Browse our providers
```

Comment:
Looks like all the text here is hard coded, how would this work users that speak French?